src/main/resources/swagger/api-docs.yaml

```diff
@@ -3534,6 +3534,36 @@ paths:
         500:
           description: Internal Server Error
       security: []
+    get:
+      tags:
+      - Entities
+      operationId: browserDownloadEntitiesTSVGet
+      summary: |
+        TSV file containing workspace entities of the specified type
+      description: |
+        swagger-ui seems to not handle file downloads, so this endpoint won't function through the ui.
+        It is here for documentation purposes only.
+      produces:
+        - text/plain
+        - application/octet-stream
+      parameters:
+        - $ref: '#/parameters/workspaceNamespaceParam'
+        - $ref: '#/parameters/workspaceNameParam'
+        - $ref: '#/parameters/entityTypeParam'
+        - name: attributeNames
+          description: comma separated list of ordered attribute names to be in downloaded tsv
+          type: string
+          in: query
+      responses:
+        200:
+          description: Workspace entities of specified type in TSV format
+          schema:
+            type: file
+        404:
+          description: Workspace or entity type does not exist
+        500:
+          description: Internal Server Error
+      security: []
```

Comment:
if the caller exceeds URL length by specifying too many `attributeNames`, what do we return? Is that a 400? Please add this potential response here in the swagger doc. (I don't think this error case needs a unit test)